tools/benchmarking.sh

```diff
@@ -5,17 +5,19 @@
 # The tanssi binary is required to be compiled with --features=runtime-benchmarks
 # in release mode.
 
-set -e
+set -euo pipefail
 
 # By default we use the tanssi-node release binary
 # However we can use any binary by running the benchmark tool with
 # BINARY=./target/release/container-chain-simple-node ./tools/benchmarking.sh
+: "${BINARY:=}"
```

Comment:
What is this?

Reply:
Previously, if benchmarking sh script is failed, the script did not fail on CI. This is why I've changed the parameter for this sh file from: 
`set -e`
to 
`set -euo pipefail`
Since we run the command in the pipe, it means that if the pipeline fails, it will fail the whole script, which is expected.

We didn't properly handle the empty parameter for both BINARY and CHAIN options, and the script was not failing because it was just ignored. Now - it is handled, and in case any of the option is not provided, it will work properly.